### ISSUE TITLE
Visualization proposal for an interactive dashboard

### ISSUE THREAD

**Issue by fccoelho** [Mosqlimate-project/visualization#1]
@aslamedeiros and I had a conversation this morning, and she made this sketch for how a dashboard could work:

![WhatsApp Image 2023-08-03 at 13 20 31](https://github.com/Mosqlimate-project/visualization/assets/140123/f69789c6-ccb2-4ce2-939c-76890f0ac587)

**Comment by fccoelho** [comment]
The Idea is that we gradually turn this into a live visualization, implemented with Altair. Let's discuss all the aspects about it here.

**Comment by fccoelho** [comment]
[Sugestão de visualização](https://fehann.github.io/VisEMAp2020Trabalho/html/chartfinal2.html) para as séries de casos

**Comment by eduardocorrearaujo** [comment]
> The Idea is that we gradually turn this into a live visualization, implemented with Altair. Let's discuss all the aspects about it here.

this package: https://altair-viz.github.io/?

**Comment by eduardocorrearaujo** [comment]
> [Sugestão de visualização](https://fehann.github.io/VisEMAp2020Trabalho/html/chartfinal2.html) para as séries de casos

Ok, vou trabalhar nessa visualização para a série de casos.

**Comment by aslamedeiros** [comment]
Referência do processo construtivo da vis de referência e link para código fonte: 
https://medium.com/datavizbr/visualizando-dados-n%C3%A3o-deixe-a-ansiedade-atrapalhar-seu-processo-criativo-aac962ad1e17

https://fehann.github.io/VisEMAp2020Trabalho/

**Comment by fccoelho** [comment]
Pensei que seria interessante adicionar uma visualização  tipo bar chart race No Epi-scanner, usando Altair já existe esta lib: 
https://www.dexplo.org/bar_chart_race/

Mas poderíamos torná-la ainda mais interessante linkando a una visualização de mapa onde os polígonos fossem acendendo à medida que as cidades vão entrando na corrida, ou seja, começam a ter casos no início da temporada epidemica.

Pensei também em explorar diferentes rankeamentos, além do tradicional por número de casos, acho que poderíamos rankear agrupar por similaridade (agglomerative clustering)

**Comment by eduardocorrearaujo** [comment]
> [Sugestão de visualização](https://fehann.github.io/VisEMAp2020Trabalho/html/chartfinal2.html) para as séries de casos

This visualization was implemented in the PR #2

**Comment by eduardocorrearaujo** [comment]
> Pensei que seria interessante adicionar uma visualização tipo bar chart race No Epi-scanner, usando Altair. Já existe esta lib: https://www.dexplo.org/bar_chart_race/
> 
> mas não suporta altair, apenas matplotlib e plotly.
> 
> Mas poderíamos torná-la ainda mais interessante linkando a una visualização de mapa onde os polígonos fossem acendendo à medida que as cidades vão entrando na corrida, ou seja, começam a ter casos no início da temporada epidemica.
> 
> Pensei também em explorar diferentes rankeamentos, além do tradicional por número de casos, acho que poderíamos rankear agrupar por similaridade (agglomerative clustering)

Criei a issue separado para esse gráfico em #4

**Comment by fccoelho** [comment]
[Sugestão de visualização](https://echarts.apache.org/examples/en/editor.html?c=map-bar-morph) para estatísticas geolocalzada baseada na lib echarts

**Comment by aslamedeiros** [comment]
Tem que pensar se essas transições animadas funcionam bem no conjunto todo.
São recursos que funcionam bem em apresentações (slides) mas são menos
úteis pra análise de informação.
São recursos atratores da atenção e portanto também distratores para um
leitor focado (que costuma ser o caso do analista).


Em qua., 27 de set. de 2023 16:27, Flávio Codeço Coelho <
***@***.***> escreveu:

**Comment by fccoelho** [comment]
I agree with your point of view @aslamedeiros , but I guess that such a visualization would be a nice addition for the Landing page of the API, for instance. As pure Eye-candy.
